conf/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cat <<EOF
+<define name="hostname" value="${HOSTNAME:-irc}">
```

Comment:
At least in insp20 this has to be an FQDN. So maybe `irc.example.com`.

Reply:
It has to be one in v3 too.